Sources/Core/AWSClientRuntime/Auth/AuthSchemes/CustomSigningPropertiesSetter.swift

```diff
@@ -0,0 +1,77 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import ClientRuntime
+
+// Service-specific signing properties customization setter.
+public class CustomSigningPropertiesSetter {
```

Comment:
As a precaution, I think the tests from the deleted `SigV4UtilTests.swift` should be migrated over.

Reply:
Just refactored the deleted tests from `SigV4Utiltests.swift` into new tests for `CustomSigningPropertiesTests.swift` as suggested.